packages/server/lib/cloud/api.ts

```diff
@@ -88,18 +87,32 @@ const rp = request.defaults((params: CypressRequestOptions, callback) => {
     if (params.encrypt === true || params.encrypt === 'always') {
       const { secretKey, jwe } = await enc.encryptRequest(params)
 
+      // TODO: double check the logic below here with @tgriesser
       params.transform = async function (body, response) {
-        if (response.headers['x-cypress-encrypted'] || params.encrypt === 'always' && response.statusCode < 500) {
+        const { statusCode } = response
+
+        // response is valid and we are encrypting
+        if (response.headers['x-cypress-encrypted'] || params.encrypt === 'always') {
           let decryptedBody
 
+          // TODO: if body is null/undefined throw a custom error
+
           try {
             decryptedBody = await enc.decryptResponse(body, secretKey)
           } catch (e) {
+            // we failed decrypting the response...
+
+            // if status code is >=500 or 404 just return body
+            if (statusCode >= 500 || statusCode === 404 || statusCode === 422) {
```

Comment:
We don't ever respond to `/preflight` with a 404 so it doesn't seem like would ever happen in real world use.

Reply:
i understand that, i'm solving for a different scenario